setup-gcloud does not work in example

it appears as though the `setup-gcloud` action in the example does not work. The workflow example cited in this repository and on the marketplace uses ubuntu-latest as the image, but its missing python-openssl from the base image as well as a setting . To make it work, this repository either needs to include it itself (probably not considering how sparse this is) _or_ you need to do the following

```
...
env:
    CLOUDSDK_PYTHON_SITEPACKAGES: 1

...

    # steps
 
    - name: install python-openssl
       run: sudo apt-get install -y python-openssl -o=Dpkg::Use-Pty=0

```

```
gcloud auth activate-service-account *** --key-file=/tmp/tmp-8931VSIBcdJY8Icv
/bin/tar xz -C /home/runner/work/_temp/c5d195bb-1ace-4331-9be7-5ddb6b45397b -f /home/runner/work/_temp/81352a93-56e4-444f-bc12-f7e7ba184cc9
WARNING: .p12 service account keys are not recomended unless it is necessary for backwards compatability. Please switch to a newer .json service account key for this account.
ERROR: (gcloud.auth.activate-service-account) PyOpenSSL is not available. If you have already installed PyOpenSSL, you will need to enable site packages by setting the environment variable CLOUDSDK_PYTHON_SITEPACKAGES to 1. If that does not work, see https://developers.google.com/cloud/sdk/crypto for details or consider using .json private key instead.
##[error]The process 'gcloud' failed with exit code 1
##[error]Node run failed with exit code 1
```

Hey sorry. I'm not trying to be obtuse about this :)

* When I navigated through the marketplace and clicked the "build and deploy to GKE" workflow I got this template https://gist.github.com/trevoro/7b2fda6fcfddb90fe858b94dc4753a5f populated in the in-browser editor
* That template has instructions that say to setup your secrets in the workspace where GKE_KEY is the service account key. I agree that there are instructions on how to format the base64 key but they're not referenced in the gist above.
* If the key is not encoded properly OR not set you get this error:
```
gcloud auth activate-service-account *** --key-file=/tmp/tmp-26506ls0Nsxf4ZHG
/bin/tar xz -C /home/runner/work/_temp/c7710d94-bd72-4383-95c1-ae8c8bd5ecd3 -f /home/runner/work/_temp/2d56b3e0-a0a1-4dfd-866a-2fc9283f36a6
WARNING: .p12 service account keys are not recomended unless it is necessary for backwards compatability. Please switch to a newer .json service account key for this account.
ERROR: (gcloud.auth.activate-service-account) PyOpenSSL is not available. If you have already installed PyOpenSSL, you will need to enable site packages by setting the environment variable CLOUDSDK_PYTHON_SITEPACKAGES to 1. If that does not work, see https://developers.google.com/cloud/sdk/crypto for details or consider using .json private key instead.
##[error]The process 'gcloud' failed with exit code 1
##[error]Node run failed with exit code 1
```
Which naturally makes one wonder why `PyOpenSSL` wouldn't be installed in the first place... presumably because its not required for the keys if they're formatted the right way.

I've gotten this working only by installing openssl for python in the workflow itself; so the base image that is the *default* template when you create the action through the marketplace seems to be missing a library required to make it work.

Can you not reproduce this on your side?

--- a/example-workflows/gke/.github/workflows/gke.yml
+++ b/example-workflows/gke/.github/workflows/gke.yml
@@ -16,7 +16,8 @@
 #
 # To configure this workflow:
 #
-# 1. Set up secrets in your workspace: GKE_PROJECT with the name of the project, GKE_EMAIL with the service account email, GKE_KEY with the service account key.
+# https://github.com/GoogleCloudPlatform/github-actions/tree/docs/service-account-key/setup-gcloud#inputs
+# 1. Set up secrets in your workspace: GKE_PROJECT with the name of the project, GKE_EMAIL with the service account email, GKE_KEY with the Base64 encoded JSON service account key.
 #
 # 2. Change the values for the GKE_ZONE, GKE_CLUSTER and IMAGE environment variables (below).
 

--- a/setup-gcloud/README.md
+++ b/setup-gcloud/README.md
@@ -53,5 +53,4 @@ steps:
 
 * `service_account_email`: (Optional) The service account email which will be used for authentication.
 
-* `service_account_key`: The service account key which will be used for authentication. This key should be stored as a
-[Base64](https://en.wikipedia.org/wiki/Base64) encoded string.
+* `service_account_key`: The service account key which will be used for authentication. This key should be [created](https://cloud.google.com/iam/docs/creating-managing-service-account-keys), encoded as a [Base64](https://en.wikipedia.org/wiki/Base64) string (eg. `cat my-key.json | base64` on macOS), and stored as a [secret](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/creating-and-using-encrypted-secrets). 

